tests/components/orderentry/AllOrders.test.jsx

```diff
@@ -23,9 +23,11 @@ describe('AllOrders test-suite', () => {
         totalCount: 20,
         links: [
           {
+            rel: 'next',
```

Comment:
What does `rel` mean? Always write variables that make the code more readable.

Reply:
@dondrzzy, this is not a variable name. Look at it again.